tests/integration/common/checks-after-conversion/test_enabled_repositories.py

```diff
@@ -28,7 +28,7 @@ def test_enabled_repositories(shell, system_release):
     enabled_repos = shell("yum repolist").output
 
     try:
-        if "redhat-8.6" in system_release:
+        if ("redhat-8.6" or "redhat-8.8") in system_release:
```

Comment:
```suggestion
        if re.match(r"redhat-8\.[68]", system_release):
```
Alternatively, we could use `re` to match those.

Reply:
Incorporated these changes in https://github.com/oamg/convert2rhel/pull/954/files